lib/vernier/output/firefox.rb

```diff
@@ -292,7 +290,6 @@ def samples_table
             times = (0...size).to_a
           end
 
-          raise unless samples.size == size
```

Comment:
I think this is redundant? `size` hasn't been re-assigned and `sample` hasn't been mutated since `size = samples.size` on line `281`.